hi-ml/src/health_ml/run_ml.py

```diff
@@ -176,9 +178,20 @@ def init_training(self) -> None:
         # Set random seeds just before training. Ensure that dataloader workers are also seeded correctly.
         seed_everything(self.container.get_effective_random_seed(), workers=True)
 
-        # get the container's datamodule
+        # Get the container's datamodule
         self.data_module = self.container.get_data_module()
 
+        # Create an AzureML run for logging if running outside AzureML. This run will be used for metrics logging
```

Comment:
Ok this answer my question above, can we precise in `log_from_vm` doc that it will log both training and inference logs to the AML run?

Reply:
Done!